prow/cmd/deck/static/spyglass/spyglass.ts

```diff
@@ -160,4 +164,27 @@ window.addEventListener('hashchange', (e) => {
 // We can't use DOMContentLoaded here or we end up with a bunch of flickering. This appears to be MDL's fault.
 window.addEventListener('load', () => {
     loadLenses();
+    handleRerunButton();
 });
+
+function handleRerunButton() {
+  const rerunStatus = getParameterByName("rerun");
+  const modal = document.getElementById('rerun')!;
+  const rerunCommand = document.getElementById('rerun-content')!;
+  window.onclick = (event: any) => {
+      if (event.target === modal) {
+          modal.style.display = "none";
+      }
+  };
+
+  const r = document.getElementsByClassName("mdl-layout__header-row")[0];
```

Comment:
Please make this more robust to unrelated changes by looking up the element in a more stable way, e.g. by ID.

Reply:
I will add an id attribute to the header element. :+1: